build.gradle

```diff
@@ -404,6 +408,29 @@ project(':iceberg-mr') {
       exclude group: 'org.apache.calcite.avatica'
     }
   }
+
+  shadowJar {
+    configurations = [project.configurations.compile]
+
+    zip64 true
+
+    // include the LICENSE and NOTICE files for the shaded Jar
+    from(projectDir) {
+      include 'LICENSE'
+      include 'NOTICE'
+    }
+
+    dependencies {
```

Comment:
More probably need to be added but the below appears to be the bare minimum needed in order to execute the table DDL and perform a simple query of a Parquet Iceberg table from Hive.